storage/onpremise/mongodb/main.tf

```diff
@@ -47,7 +47,19 @@ resource "helm_release" "mongodb" {
       "auth" = {
         "databases" = ["database"]
       }
-
+      "initdbScripts" = {
```

Comment:
I thought there were some options in the helm release to handle this kind of use-cases.
@tschneider-aneo What do you think?

Reply:
A user can be created out-of-the box by adding a value in `auth.usernames` , `auth.passwords` options but also in `auth.databases`. This latter option adds custom databases at MongoDB init. These three options are actually arrays that have to be of the same length (1 user per DB).

IMO, this is a weird way for the Helm chart to create DB's and users so maybe I should post another issue on Bitnami's charts repo. Anyway, in this case `admin` is not a custom DB so I don't think it should be added to the `auth.databases` option.

I don't know if [RBAC-related options](https://artifacthub.io/packages/helm/bitnami/mongodb#rbac-parameters) could work for defining particular roles in a DB, it could be tried (and could also be in a feature request in Bitnami's charts repo).

Finally, I don't think we should have this script directly written in `main.tf`, I would prefer to have it in another file and then referenced in `main.tf`.

Reply:
No issues with putting the script in another file.
Regarding RBAC options for the Helm chart, I don't think it is related. I have the impression it is more about creating Kubernetes Service Account.